apps/Typescript-ObjC/App.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React, {useEffect} from 'react';
+import type {PropsWithChildren} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  Button,
+  useColorScheme,
+  View,
+} from 'react-native';
+
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+import {
+  CustomerIO,
+  CustomerIOEnv,
+  CustomerioConfig,
+  CioLogLevel,
+} from 'customerio-reactnative';
+
+type SectionProps = PropsWithChildren<{
+  title: string;
+}>;
+
+function Section({children, title}: SectionProps): JSX.Element {
+  const isDarkMode = useColorScheme() === 'dark';
+  return (
+    <View style={styles.sectionContainer}>
+      <Text
+        style={[
+          styles.sectionTitle,
+          {
+            color: isDarkMode ? Colors.white : Colors.black,
+          },
+        ]}>
+        {title}
+      </Text>
+      <Text
+        style={[
+          styles.sectionDescription,
+          {
+            color: isDarkMode ? Colors.light : Colors.dark,
+          },
+        ]}>
+        {children}
+      </Text>
+    </View>
+  );
+}
+
+function App(): JSX.Element {
+  useEffect(() => {
+    const env = new CustomerIOEnv();
+    env.siteId = 'YourSiteId';
+    env.apiKey = 'YourAPIKey';
+
+    const config = new CustomerioConfig();
+    config.logLevel = CioLogLevel.debug;
+
+    CustomerIO.initialize(env, config);
+  }, []);
+
+  const isDarkMode = useColorScheme() === 'dark';
```

Comment:
I see this isDarkMode const being declared twice and didn't expect that. Is this correct or should it have been declared once?

Reply:
Unfortunately, I don't know enough RN to know if this was needed or not. This code was generated from the community template so I am unsure. 

This template project does include some optional code in it that's probably not needed - including dark mode. Since none of our other sample apps include dark mode in it, I think this sample app should match that.